.github/workflows/main.yml

```diff
@@ -94,7 +69,9 @@ jobs:
 
     - name: Build StarlingMonkey WPT
       run: |
-        cmake -S . -B cmake-build-${{ matrix.build }} -DENABLE_WPT:BOOL=ON -DCMAKE_BUILD_TYPE=${{ matrix.build == 'release' && 'Release' || 'Debug' }}
+        cmake -S . -B cmake-build-${{ matrix.build }} -DENABLE_WPT:BOOL=ON\
+          -DCMAKE_BUILD_TYPE=${{ (matrix.build == 'release' || matrix.build == 'weval') && 'Release' || 'Debug' }}\
+          ${{matrix.build == 'weval' && '-DUSE_WASM_OPT=OFF -DWEVAL=ON' || ''}}
```

Comment:
I think the failure here (a missing "database file") is the wevalcache file not being downloaded; it's a separate artifact IIRC?

Reply:
This workflow does the build and test run in the same task, so it should be there.

I think the issue is we need the WPT plumbing here for the WPT build script like in js-compute-runtime to run Weval componentize properly?

Reply:
I've got to run just now but I was able to get this to build locally by building the `starling-ics.wevalcache` target first: `cmake --build cmake-build --parallel 4 starling-ics.wevalcache` then `cmake ... wpt-runtime`. Perhaps there's a way to get the dependencies right to handle that automatically, or otherwise we can just add the extra target here?